numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -4088,6 +4088,7 @@ initialize_numeric_types(void)
      * #NAME = Byte, Short, Int, Long, LongLong, UByte, UShort,
      *         UInt, ULong, ULongLong#
      */
+    @name@_arrtype_as_number = gentype_as_number;
```

Comment:
FYI as a rule of thumb, when replying to github messages from your email client then you're best off top-posting, because github's email parser is kinda terrible. (See the github post that this is replying to for an example.)